src/app/App.js

```diff
@@ -7,33 +7,26 @@ import PropTypes from 'prop-types';
 
 import isEqual from 'lodash/isEqual';
 import { connect } from 'react-redux';
-import { getParticipantList, setRandomParticipant, removeParticipant } from '../actions/participants';
+import { getParticipantList, setRandomParticipant, removeParticipant, fileUpload } from '../actions/participants';
 
 //Components
 import SearchComponent from '../components/searchComponent/searchComponent';
 import ListOfParticipants from '../components/listOfParticipants/listOfParticipants';
-import ParticipantCard from '../components/listOfParticipants/participantCard/participantCard';
-
-//material-components
-import Button from '@material-ui/core/Button';
-import DeleteIcon from '@material-ui/icons/Delete';
+import SelectedCard from '../components/selectedCard/selectedCard';
+import UploadFile from '../components/uploadFile/uploadFile';
 
 class App extends Component {
 
     componentDidMount() {
-        this.props.getParticipantList();
+        // this.props.getParticipantList();
```

Comment:
If this change is intentional (as I think it's), please remove the componentDidMount hook